docs/data/date-pickers/custom-field/MultiInputFieldSeparatorSlotProps.js

```diff
@@ -8,7 +8,7 @@ export default function MultiInputFieldSeparatorSlotProps() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <DemoContainer components={['DateRangePicker', 'DateRangePicker']}>
-        <DateRangePicker slotProps={{ fieldSeparator: { children: 'to' } }} />
+        <DateRangePicker slotProps={{ fieldSeparator: { sx: { color: 'red' } } }} />
```

Comment:
I kept this demo but I guess most people only used it to change the separator
Do you think we should remove it?
Do you see other props we could pass to the typography here?

Reply:
But I struggle to find a good top-level prop usage
What does `variant: 'span'` do? :laughing: 
When I have no idea, I usually just add an id or a data-testid because it's super generic, but for the field separator it's not a very interesting customization :grimacing:

Reply:
> What does `variant: 'span'` do? 😆

It changes the default `p` element to a `span`. 😉 
But it seems that such an example is not the most correct [props-wise](https://mui.com/material-ui/api/typography/#props). 🙈 
Maybe `component: 'span'` or `variant: 'body1'`? 🤔

Reply:
:thinking: But `span` is not a valid variant (unless [this list is incomplete](https://mui.com/material-ui/react-typography/#changing-the-semantic-element))

But in any case, I can change the variant :+1:

Reply:
Done